jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenLayerConfigurations.java

```diff
@@ -71,7 +90,7 @@ static JavaLayerConfigurations getForProject(
     Path classesOutputDirectory = Paths.get(project.getBuild().getOutputDirectory());
 
     // Gets the classes files in the 'classes' output directory.
-    Predicate<Path> isClassFile = path -> path.toString().endsWith(".class");
+    Predicate<Path> isClassFile = path -> path.toString().endsWith(CLASS_EXTENSION);
```

Comment:
nit: I'm fine with embedding the string `.class`.

Reply:
Done